adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -1233,4 +1193,66 @@ private static void addRetainAnnotation(final MqttContext context, final Message
             MessageHelper.addAnnotation(downstreamMessage, MessageHelper.ANNOTATION_X_OPT_RETAIN, Boolean.TRUE);
         }
     }
+
+    private Future<TenantObject> isAdapterEnabled(final DeviceCredentials credentials, final TenantObject tenantConfig) {
```

Comment:
There is an already existing debug statement that prints authId and only for that sake, credentials parameter is needed

Reply:
hm, I do not seem to be able to find the location, can you help me out?

Reply:
I referred to `isDeviceRegistered` instead of `isAdapterEnabled`. As you suggested, the ID of the tenantConfig object can be used instead from the credentials parameter.